src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/TargetsTests.cs

```diff
@@ -180,4 +180,23 @@ public void CanComputeContainerUser(string tfm, string rid, string expectedUser)
         var computedTag = instance.GetProperty("ContainerUser")?.EvaluatedValue;
         computedTag.Should().Be(expectedUser);
     }
+
+    [InlineData("linux-x64", "linux-x64")]
+    [InlineData("linux-arm64", "linux-arm64")]
+    [InlineData("windows-x64", "linux-x64")]
+    [InlineData("windows-arm64", "linux-arm64")]
+    [Theory]
+    public void WindowsUsersGetLinuxContainers(string sdkPortableRid, string expectedRid)
+    {
+        var (project, logger, d) = ProjectInitializer.InitProject(new()
+        {
+            ["TargetFrameworkVersion"] = "v6.0",
+            ["NETCoreSdkPortableRuntimeIdentifier"] = sdkPortableRid
+        }, projectName: $"{nameof(WindowsUsersGetLinuxContainers)}_{sdkPortableRid}_{expectedRid}");
```

Comment:
very minor: though syntax is correct, the code does not feel very csharpie. Feels like some functional program written in c#.  A respectful opinion, not a fact.